specification/hybridconnectivity/resource-manager/readme.md

```diff
@@ -27,9 +27,17 @@ These are the global settings for the hybridconnectivity.
 ``` yaml
 openapi-type: arm
 openapi-subtype: rpaas
-tag: package-2023-03
+tag: package-2024-12-01
 ```
 
+### Tag: package-2024-12-01
+
+These settings apply only when `--tag=package-2024-12-01` is specified on the command line.
+
+```yaml $(tag) == 'package-2024-12-01'
+input-file:
+  - Microsoft.HybridConnectivity/PublicCloud/stable/2024-12-01/publicCloud.json
```

Comment:
Please add **Microsoft.HybridConnectivity/stable/2023-03-15/hybridconnectivity.json** too in the input file.

Please check [azure-rest-api-specs-pr branch](https://github.com/Azure/azure-rest-api-specs-pr/blob/RPSaaSMaster/specification/hybridconnectivity/resource-manager/readme.md?plain=1#L88-L96) change done by @PriyankaHariharan 

This will resolve your **_Swagger Avocado_** failure